.github/workflows/deployment.yml

```diff
@@ -3,17 +3,37 @@ name: Deployment
 on: [push]
 
 jobs:
-  deploy:
+  # In case the timestamp changes, much data is synched by rsync
```

Comment:
Don't understand...timestamps of what?

Reply:
See my full answer and I also refined the comment (still working on the implementation)